src/YGPushButton.cc

```diff
@@ -9,6 +9,7 @@
 #include "YGUtils.h"
 #include "YGWidget.h"
 #include <string.h>
+#include <libgen.h>
```

Comment:
I haven't experienced any really... but in the case we could use the boost methods to manage filename and dirname... is a bit complex i think...